spec.html

```diff
@@ -50186,11 +50186,11 @@ <h1>String.prototype.fixed ( )</h1>
       </emu-annex>
 
       <emu-annex id="sec-string.prototype.fontcolor">
-        <h1>String.prototype.fontcolor ( _color_ )</h1>
+        <h1>String.prototype.fontcolor ( _colour_ )</h1>
         <p>This method performs the following steps when called:</p>
         <emu-alg>
           1. Let _S_ be the *this* value.
-          1. Return ? CreateHTML(_S_, *"font"*, *"color"*, _color_).
+          1. Return ? CreateHTML(_S_, *"font"*, *"color"*, _colour_).
```

Comment:
Let's keep these _color_. I'd prefer algorithm steps and aliases to be American spelling, like code.